.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,102 @@
+---
+name: Bug report
+description: Create a BUG report.
+title: "[BUG]: Title"
+labels: Bug
+body:
+- type: markdown
+  attributes:
+    value: |
+      Before you submit this request, please be aware that this is an
```

Comment:
Please add that before asking reporting bug feature or question, reporter should try latest upstream.

I would like to filter OSv related questions as this is not in our control.

Reply:
There shouldn't be os dependent issues, ledmon is simple.